.github/workflows/codeql.yml

```diff
@@ -0,0 +1,51 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
```

Comment:
PRs are usually merged to `dev` and not to `master`. Should we change it to `dev` and change `push` to both `master` and `dev`?

Reply:
I think `dev` is enough.